src/explore-education-statistics-admin/src/styles/patterns/_dfe-modifiers.scss

```diff
@@ -114,6 +114,10 @@ h1.govuk-panel__title {
   padding: 0 6px 6px;
   position: fixed;
   z-index: 10000;
+
+  @media print {
```

Comment:
I've got rid of `dfe-modifiers.scss` in admin and placed this particular style block in `EditablePageModeToggle.module.scss`. I would rebase this branch on top of EES-629